.goreleaser.yml

```diff
@@ -206,6 +207,9 @@ nfpms:
         dependencies:
           - passwd
           - ca-certificates
+          - bee-clef
```

Comment:
does this mean that bee-clef becomes a dependency for deb+rpm packages?

Reply:
@acud as I asked [here](https://beehive.ethswarm.org/swarm/pl/7x8dzfryuirhib57x13iisg7gh)
This PR will wait for us to decide in which direction to go